source/_integrations/pi_hole.markdown

```diff
@@ -63,3 +63,7 @@ data:
 target:
   entity_id: all
 ```
+
+## Switches
+
+The integratation will create a switch for the pi-hole named `switch.pi_hole` which will allow you to toggle the pi-hole on and off.
```

Comment:
Wait, my review comment was: Remove the entity ID completely?

But it is still in?

../Frenck

Reply:
thank you for the catch, @frenck 
#31424